corim/example_profile_test.go

```diff
@@ -195,5 +195,5 @@ func Example_profile_marshal() {
 	fmt.Printf("corim: %v", hex.EncodeToString(buf))
 
 	// output:
-	// corim: d901f5a300f6018158d9d901faa40065656e2d474201a100676578616d706c650281a4006941434d45204c74642e01d8207468747470733a2f2f61636d652e6578616d706c65028101206f3132332046616b652053747265657404a1008182a100a300d90258582061636d652d696d706c656d656e746174696f6e2d69642d303030303030303031016941434d45204c74642e026e526f616452756e6e657220322e3081a200d90259a30162424c0465352e302e35055820acbb11c7e4da217205523ce4ce1a245ae1a239ae3c6bfd9e7871f7e5d8bae86b01a10281820644abcdef00037822687474703a2f2f6578616d706c652e636f6d2f6578616d706c652d70726f66696c65
+	// corim: d901f5a300f60181d901fa58d6a40065656e2d474201a100676578616d706c650281a4006941434d45204c74642e01d8207468747470733a2f2f61636d652e6578616d706c65028101206f3132332046616b652053747265657404a1008182a100a300d90258582061636d652d696d706c656d656e746174696f6e2d69642d303030303030303031016941434d45204c74642e026e526f616452756e6e657220322e3081a200d90259a30162424c0465352e302e35055820acbb11c7e4da217205523ce4ce1a245ae1a239ae3c6bfd9e7871f7e5d8bae86b01a10281820644abcdef00037822687474703a2f2f6578616d706c652e636f6d2f6578616d706c652d70726f66696c65
```

Comment:
Not for this PR necessarily (as the bug was there before), but we have a problem with this CoRIM: it has a `null` tag-id, which is invalid.  This should’ve been caught by `Valid()`, no?

Reply:
no, wait, I've confused with `comid`. You're right. I'm not sure what is happening here...

Reply:
Ok, The issue is that `UnsignedCorim.ToCBOR` doen't validate. I'll submit that as a separate fix.